main.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"github.com/dannegm/academy-go-q12021/controller"
+	"github.com/dannegm/academy-go-q12021/models"
+	"github.com/gin-gonic/gin"
```

Comment:
As a best practice, we should organize our imports in alphabetic order and taking first into consideration the following groups separated by an empty line:

First, Go's standard libraries
Second, your local packages
And last, 3rd party packages

Something like this:
```suggestion
import (
	"github.com/dannegm/academy-go-q12021/controller"
	"github.com/dannegm/academy-go-q12021/models"
	
	"github.com/gin-gonic/gin"
```

Please, apply these changes to all other project's occurrences.

Reply:
These are auto-ordered, with the go tools I guess, I've tried to put in an alphabetic order before :(

Reply:
There are some formaters that do the trick for you;

Give it a look to gofmt, goimports, goformat, gofumpt.

I use gofmt.